.github/actions/backup-restore-test/action.yml

```diff
@@ -86,10 +92,10 @@ runs:
         ./sanity-test
 
     - name: Export ${{ inputs.service }} ${{ inputs.kind }}
+      if: ${{ inputs.with-export == true }}
```

Comment:
Nit: do we need the `== true` check?

Reply:
probably not.  But it's not really a bug.  Just excess.